### DISCUSSION TITLE
Copilot autocompletes all suggestion, all lines with XML comment start (///) in C#

### DISCUSSION THREAD

**Discussion by r-pankevicius** [community/community#55842] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

I'm using Github Copilot in Visual Studio 2022, latest:
Microsoft Visual Studio Professional 2022 (64-bit) - Current
Version 17.5.5

it seems I've trained Copilot too much to shorted XML comments in one line style (they typically take 3 lines).
I am using it for simple properties in simple data contracts, like:
```csharp
public class Person
{
    /// <summary>Full name</summary>
    public string Name { get; set; }

    /// <summary>Age in years</summary>
    public int Age { get; set; }
}
```
Now I have issue with Copilot auto completion. The suggestion is quite OK:
![image](https://github.com/community/community/assets/1467109/ee8ab4cb-2225-4604-880f-c3b2d8afdeb8)
But when I press tab (I call it autocompletion, not sure if that's term you use), I expect the suggestion will be typed in intact, but all that is commented twice starting from the second line:
```csharp
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Mynamespace
{
    /// <summary>
    /// Selected value in the combobox: int key and string value.
    /// </summary>
    public class ComboboxValue
    {
        /// <summary>Key.</summary>
        ///        public int Key { get; set; }
        ///               /// <summary>Value.</summary>
        ///                      public string Value { get; set;
    }
}
```

**Comment by iSinner1** [comment]
I have the same issue. And pressing ctr+z to undo, removes all generated code, signifying that there is no formatting happening from another tool after copilot code generation, so this looks like a copilot thing and not some other interference. Very annoying.

**Comment by AlexSherstyuk** [comment]
Encounter same problem: GH Copilot is not handling well /// comments for C#.

**Reply by ayim** [reply]
Hey! Could you confirm your VS & Copilot extension version and provide a code sample?
We're leaning heavy into quality in the coming weeks.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
github:ayim | aayim@microsoft.com

**Comment by AlexSherstyuk** [comment]
Hi Aaron,

Please find attached ProblemDescription.zip with code sample, GH Copilot log and other information.
By the way, it appears practically impossible to submit bug report via GH support page: attachments are rejected.

Regards,
   AlexS
[ProblemDescription.zip](https://github.com/community/community/files/14473772/ProblemDescription.zip)

**Reply by AlexSherstyuk** [reply]
P.S.
  Finally I've managed to submit Ticket #2647458 with ProblemDescription.zip attached.
It appears that files cannot be attached to the main body of the bug report, but attachment is accepted with comment added later to the ticket.

**Reply by ayim** [reply]
Hi Alex, thanks for repro-ing and sending examples over.
Acknowledging we've got em and I'm working with eng to investigate.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
gh: ayim | aayim@microsoft.com
